src/App.css

```diff
@@ -0,0 +1,7 @@
+.delete-button{
```

Comment:
I'd suggest configuring the IDE linter/formatter, see that there's no space between `button` and `{`